gateleen-security/src/main/java/org/swisspush/gateleen/security/authorization/RoleMapper.java

```diff
@@ -0,0 +1,89 @@
+package org.swisspush.gateleen.security.authorization;
+
+import io.vertx.core.buffer.Buffer;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.swisspush.gateleen.core.storage.ResourceStorage;
+import org.swisspush.gateleen.validation.ValidationException;
+
+import java.util.*;
+
+public class RoleMapper implements ConfigurationResource {
+
+    private ResourceStorage storage;
+
+    private String roleMapper;
+
+    private RoleMapperFactory roleMapperFactory;
+
+    private List<RoleMapperHolder> roleMappers = null;
+
+    public static final Logger log = LoggerFactory.getLogger(RoleMapper.class);
+
+    public RoleMapper( ResourceStorage storage, String securityRoot) {
```

Comment:
Please add JavaDoc, espacially to clarify parameter `securityRoot`.
Note that you implicit expect a trailing-slash - but you don't enforce it. So at lease document it

Reply:
The method should now be smart enough to concat the right way no matter what kind of slash patterns are provided.
Added JavaDoc